integrations/aws/session_manager.py

```diff
@@ -0,0 +1,134 @@
+import os
+from typing import Any
+import aioboto3
+from aws_credentials import AwsCredentials
+from port_ocean.context.ocean import ocean
+from loguru import logger
+
+
+class SessionManager:
+    def __init__(self):
+        self._aws_accessible_accounts: list[dict[str, Any]] = []
+        self._aws_credentials: list[AwsCredentials] = []
+    
+    async def reset(self):
+        application_credentials = await self._get_application_credentials()
+        await application_credentials.update_enabled_regions()
+        self._application_account_id = application_credentials.account_id
+        self._application_session = await application_credentials.create_session()
```

Comment:
Instance attribute _application_account_id defined outside __init__  please fix in other places 2